toe/src/tile/tile.js

```diff
@@ -301,26 +303,29 @@ export default class toeTile extends Component {
                 squares[data.move[0] - 1][data.move[1] - 1] = data.stone;
               }
             }
-            if (data.status === "replay") {
-              game = "replay";
+            if (data.status === "rematch") {
+              console.log("rematching");
```

Comment:
There's a console.log here, and a couple lines down. Might be nice to clean that up.